src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertTo-Html.cs

```diff
@@ -198,6 +198,27 @@ public SwitchParameter Fragment
 
         private SwitchParameter _fragment;
 
+        /// <summary>
+        /// When this switch is specified auto
+        /// generate hyperlinks for URI.
+        /// </summary>
+        [Parameter(ParameterSetName = "Hyperlink")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter Hyperlink
+        {
+            get
+            {
+                return _hyperlink;
+            }
+
+            set
+            {
+                _hyperlink = value;
+            }
+        }
+        
+        private SwitchParameter _hyperlink;
```

Comment:
Please use auto property.

Reply:
I changed it to bool since this is ment to be a switch